src/renderer/components/ft-playlist-selector/ft-playlist-selector.scss

```diff
@@ -0,0 +1,280 @@
+$thumbnail-overlay-opacity: 0.85;
+$watched-transition-duration: 0.5s;
```

Comment:
**issue (non-blocking):** Many of the styling rules in this file seem to not actually be matching anything. I would say it's copied from somewhere, but the styling seems to be unique in some parts for reasons I can't discern (e.g., the `videoDuration` class rules).

Reply:
If at least some of this is important but is just rote copying, my suggestion would be to create a `mixin` for that and just import it here.

Reply:
It's too early to call for style variable extraction
Since after removing unused style they become unused